aws/vpc/README.md

```diff
@@ -4,46 +4,50 @@ be useful.
 
 ## What this does
 
- - Create VPC named after `app_name` and `app_env`
- - Create public and private subnets for each `aws_zones` specified
- - Provision a Internet Gateway and configure public subnets to route through it
- - Provision a NAT Gateway (or use an existing Transit Gateway) and configure private subnets to route through it
- - Create a DB subnet group including all private subnets
+- Create VPC named after `app_name` and `app_env`
+- Create public and private subnets for each `aws_zones` specified
+- Provision a Internet Gateway and configure public subnets to route through it
+- Provision a NAT Gateway (or use an existing Transit Gateway) and configure private subnets to route through it
+- Create a DB subnet group including all private subnets
+- Optionally allocate IPv6 CIDR blocks, egress-only internet gateway, and default IPv6 routes
 
 ## Required Inputs
 
- - `app_name` - Name of application, ex: Doorman, IdP, etc.
- - `app_env` - Name of environment, ex: prod, test, etc.
- - `aws_zones` - A list of zones to create subnets in, ex: `["us-east-1c", "us-east-1d", "us-east-1e"]`
+- `app_name` - Name of application, ex: Doorman, IdP, etc.
```

Comment:
Minor: Do we want to continue mentioning Doorman? Suggest using `Cover` or `Ethnologue` instead.